package.json

```diff
@@ -29,6 +29,7 @@
     "prettier": "prettier --check .",
     "prettier-write": "prettier --write .",
     "serve": "vite preview",
+    "start": "vite --open",
```

Comment:
this is already the `dev-app` script

Reply:
Yes I know. You would like to keep 'start' for another purpose ? We use it in NMRium as well and I thought it is practical to simply enter `npm start` to test the full application.

Reply:
I don't like `start` for development. In the ecosystem it is used to start the application in production